src/Ocelot/Request/Mapper/StreamHttpContent.cs

```diff
@@ -8,25 +8,24 @@ public class StreamHttpContent : HttpContent
     private const int DefaultBufferSize = 65536;
     public const long UnknownLength = -1;
     private readonly HttpContext _context;
+    private readonly long _contentLength;
 
     public StreamHttpContent(HttpContext context)
     {
         _context = context ?? throw new ArgumentNullException(nameof(context));
+        _contentLength = context.Request.ContentLength ?? UnknownLength;
     }
 
-    protected override async Task SerializeToStreamAsync(Stream stream, TransportContext context,
-        CancellationToken cancellationToken)
-        => await CopyAsync(_context.Request.Body, stream, Headers.ContentLength ?? UnknownLength, false,
-            cancellationToken);
+    protected override async Task SerializeToStreamAsync(Stream stream, TransportContext context, CancellationToken cancellationToken)
+        => await CopyAsync(_context.Request.Body, stream, _contentLength, false, cancellationToken);
 
     protected override async Task SerializeToStreamAsync(Stream stream, TransportContext context)
-        => await CopyAsync(_context.Request.Body, stream, Headers.ContentLength ?? UnknownLength, false,
-            CancellationToken.None);
+        => await CopyAsync(_context.Request.Body, stream, _contentLength, false, CancellationToken.None);
```

Comment:
`await` can be removed!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1972/commits/dcbbab7119180345afaf001fbf3664f64b2fafd7 ✔️